### DISCUSSION TITLE
[RFC] can i add performer,studio,etc to my database from filenames?

### DISCUSSION THREAD

**Discussion by traplover4200** [stashapp/stash#4445] [Q&A]
*### Scope*
automatically read filenames and add performer,studio,etc to my database
if it find a studio name in the filename it will try to add it to my list of studio
if it find a performer name in the filename it will try to add it to my list of performer

## Examples
FamilyTherapyXXX - Goth Girl Dating Guide - 2022-09-25 - Alex Adams Valerica Steele - WEBDL-720p
Futanari XXX - Futa Reality - 2023-01-06 - Josephine Jackson - DVD
Horror Porn - Roswell Ufo - 1970-01-01 - - WEBDL-480p

**Comment by feederbox826** [comment]
You can use the batch-add endpoints https://github.com/stashapp/stash/pull/6310
